src/index.ts

```diff
@@ -469,6 +470,7 @@ export function create (rawOptions: CreateOptions = {}): Register {
 
       // Create the compiler host for type checking.
       const serviceHost: _ts.LanguageServiceHost = {
+        getProjectVersion: () => `${ projectVersion }`,
```

Comment:
Nit: Personally I'd just do `String(projectVersion)` since you just want to type cast.